tests/udf/test_simple_udf.py

```diff
@@ -358,16 +358,16 @@ def process_frame(self, frame):
         pass
 
 
-def test_invalid_extra_shape(lt_ctx):
+def test_extra_shape_with_zero(lt_ctx):
     data = _mk_random(size=(16, 16, 16, 16), dtype="float32")
     dataset = MemoryDataSet(data=data, tileshape=(1, 16, 16),
                             num_partitions=2, sig_dims=2)
 
     udf = InvalidBuffer()
```

Comment:
I think the UDF and the result buffer name should be renamed now, too. Maybe also add result buffers for `kind="sig"` and `kind="single"` to have all buffer kinds tested at a single place - if you like, you can make separate test functions for each buffer kind, but one UDF should be enough.